### PR TITLE
[#1983] Fix failing moveFrameLeft test

### PR DESCRIPTION
**Pull request by dseif**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/fc635de3a19b5e6/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/fc635de3a19b5e6/output.txt

Total script time: 0.98 mins

Server is running on http://pr777.butyr.org/templates/basic/
Run the Test Suite: http://pr777.butyr.org/test/

File: test/trackevent/trackevent-moveFrameLeft-zero.html [hunk 1]
```diff
@@ -13,9 +13,11 @@
         createButterModule( function( butter ) {
           var te = butter.currentMedia.tracks[ 1 ].trackEvents[ 0 ],
               inc = 3,
+              minimumTrackEventSize = 0.2,
               oldStart = te.popcornOptions.start,
               oldEnd = te.popcornOptions.end;
 
+              console.log( oldEnd, inc );
```
Comment (mjschranz) [inline test/trackevent/trackevent-moveFrameLeft-zero.html]: Remove the console.log please.

**Comment by jbuck** [comment]
Staged on master: https://github.com/mozilla/butter/commit/5bd0e6e0ad0d214148c69ae474463d0968b8b64f

**Pull request closed without merging**